Lib/test/test_pyclbr.py

```diff
@@ -5,8 +5,10 @@
 
 import sys
 from textwrap import dedent
+from contextlib import contextmanager
 from types import FunctionType, MethodType, BuiltinFunctionType
 import pyclbr
+import importlib.machinery
```

Comment:
Let's put this import before `import sys` (I know that `pyclbr` and following imports are not sorted, but let's make them sorted for now)